webui_runner/plugs/lucet_app/echo_commit.sh

```diff
@@ -0,0 +1,5 @@
+ORIG_CWD="$(pwd)"
+SCRIPT_LOC="$(dirname ${BASH_SOURCE:-$0})"
+cd ${SCRIPT_LOC}/build/lucet/
+echo $(git log -1 --pretty=format:"%h")
+cd ${ORIG_CWD}
```

Comment:
I think this could be reduced to `git --git-dir="$SCRIPT_LOC/build/lucet/.git" log -1 --pretty=format:"%h"`